### DISCUSSION TITLE
SplitChunksplugir Cache group "taro"conflicts with existing chunk,升级了taro到3.6.10，webpack升级到5.78.0，报错了?

### DISCUSSION THREAD

**Discussion by Lawrence51** [NervJS/taro#14938] [Q & A]
![企业微信截图_17017822423129](https://github.com/NervJS/taro/assets/20532261/73c7777d-5913-4134-85a9-3be62229a4e7)
目前两个页面，如果去除其中一个页面 @tarojs/components 的引入，就会好

**Comment by Lawrence51** [comment]
目前属于在老项目上进行升级，老项目使用的是原生的微信小程序，后来引入了vue，写了几个页面，现在想换成react，然后在引包时就出现了如上问题，请问各位有遇到的吗？

**Comment by xiaobc1234** [comment]
我也遇到同样的问题，最后解决了吗？

**Comment by xiaobc1234** [comment]
问题解决了，app.config.ts中路由出现了重复的路由。
![image](https://github.com/user-attachments/assets/b08426f1-c0e1-49c6-aa73-a97c8e5a3aa5)
将入口页面放到pages里面就可以了
![image](https://github.com/user-attachments/assets/47df8f55-79ac-49b0-a838-40f3803e65b0)
